Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM node:boron
 
 # Update and install packages
 RUN apt-get update && \
-    apt-get install -y python python-dev python-pip python-virtualenv zip rsync && rm -rf /var/lib/apt/lists/*
+    apt-get install -y python python-dev python-pip python-virtualenv zip rsync default-jdk && \
```

Comment:
JDK will get installed correctly here. All you need to do is set the `JAVA_HOME` env var properly and add `$JAVA_HOME/bin` to your `PATH`